src/github/utils/config.ts

```diff
@@ -2,10 +2,51 @@ import { Value } from "@sinclair/typebox/value";
 import YAML from "yaml";
 import { GitHubContext } from "../github-context";
 import { expressionRegex } from "../types/plugin";
-import { configSchema, PluginConfiguration } from "../types/plugin-configuration";
+import { configSchema, configSchemaValidator, PluginConfiguration } from "../types/plugin-configuration";
 import { eventNames } from "../types/webhook-events";
 
 const UBIQUIBOT_CONFIG_FULL_PATH = ".github/.ubiquibot-config.yml";
+const UBIQUIBOT_CONFIG_ORG_REPO = "ubiquibot-config";
+
+async function getConfigurationFromRepo(context: GitHubContext, repository: string, owner: string) {
+  const targetRepoConfiguration: PluginConfiguration = parseYaml(
+    await download({
+      context,
+      repository,
+      owner,
+    })
+  );
+  if (targetRepoConfiguration) {
+    try {
+      const configSchemaWithDefaults = Value.Default(configSchema, targetRepoConfiguration) as Readonly<unknown>;
+      const errors = configSchemaValidator.testReturningErrors(configSchemaWithDefaults);
```

Comment:
why is this needed?

Reply:
Not needed, just a nice to have to describe precisely what is wrong in the configuration. Without it, the only error I had was `Error decoding value`, which doesn't point me to the error. With this, it gives you specifically what key is wrong and what type is expected, such as:
```
{
  type: 6,
  schema: {
    minItems: 1,
    default: [],
    type: "array",
    items: {
      type: "object",
      properties: [Object ...],
      required: [ "plugin", "type", "with" ],
      [Symbol(TypeBox.Kind)]: "Object",
    },
    [Symbol(TypeBox.Kind)]: "Array",
  },
  path: "/plugins/*/1/uses",
  value: null,
  message: "Expected array",
}

```